css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,72 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  text-align: center;
+  width: 1200px;
+  font-family: 'DM Sans', sans-serif;
+}
+
+img {
+  height: 250px;
+  width: 200px;
+}
+
+#wrapper {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.container1 {
+  width: 20%;
+}
+
+.size {
+  color: gray;
+  text-decoration: line-through;
+}
+
+.size-top {
+  color: gray;
+}
+
+.dollar {
+  background-color: rgb(253, 238, 240);
+  font-weight: bold;
+  font-size: 12px;
+}
+
+.dollar20 {
+  margin-left: 2rem;
+}
+
+.cost-item-1 {
+  padding-right: 2rem;
+}
+
+.cost-item-2 {
+  padding-right: 5rem;
+}
+
+.cost-item-3 {
+  padding-right: 4rem;
+}
+
+.cost-item-4 {
+  padding-right: 6.5rem;
+}
+
+.tiffany {
+  padding-right: 4rem;
+}
```

Comment:
These should not be necessary. Probably will be fixed by not centering all the text.